test/unit/org/apache/cassandra/cql3/validation/operations/CreateTest.java

```diff
@@ -681,25 +681,17 @@ public void testCreateTableWithCompression() throws Throwable
         assertSchemaOption("compression", map("chunk_length_in_kb", "32", "class", "org.apache.cassandra.io.compress.SnappyCompressor"));
 
         createTable("CREATE TABLE %s (a text, b int, c int, primary key (a, b))"
-                + " WITH compression = { 'sstable_compression' : 'SnappyCompressor', 'chunk_length_kb' : 32 };");
-        assertSchemaOption("compression", map("chunk_length_in_kb", "32", "class", "org.apache.cassandra.io.compress.SnappyCompressor"));
```

Comment:
We should keep it as well, right?

Reply:
I think it is a minor comment, the assertion works correctly now. 
I have just thought that `"chunk_length_in_kb", 32` should be added instead of `'chunk_length_kb' : 32`